.eslintrc

```diff
@@ -2,5 +2,8 @@
   "env": {
     "node": true
   },
-  "extends": "airbnb-base"
+  "extends": ["airbnb-base", "plugin:prettier/recommended"],
```

Comment:
@ewolfe This makes the formatting consistent when you run lint and also avoids airbnb's complaints about formatting rules.